tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -319,32 +291,32 @@ df_pca[!, :type] = repeat([1, 2]; inner=n_cells ÷ 2)
 
-A direct question arises from above practice is: how many principal components do we want to keep in order to sufficiently represent the latent structure in the data?
+A direct question arises from above practice is: how many principal components do we want to keep, in order to sufficiently represent the latent structure in the data?
 This is a very central question for all latent factor models, i.e. how many dimensions are needed to represent that data in the latent space.
-In the case of PCA, there exist a lot of heuristics to make that choice. For example, We can tune the number of principal components using empirical methods such as cross-validation based some criteria such as MSE between the posterior predicted $\hat{x}$ and $x$ or the percentage of variation explained [3].
+In the case of PCA, there exist a lot of heuristics to make that choice. For example, We can tune the number of principal components using empirical methods such as cross-validation based some criteria such as MSE between the posterior predicted (e.g. mean predictions) data matrix and the original data matrix or the percentage of variation explained [3].
 
-For p-PCA, this can be done in an elegant and principled way, suing a technique called *Automatic Relevance Determination*(ARD).
+For p-PCA, this can be done in an elegant and principled way, using a technique called *Automatic Relevance Determination* (ARD).
 ARD can help pick the correct number of principal directions by regularizing the solution space using a parameterized, data-dependent prior distribution that effectively prunes away redundant or superfluous features [4].
-Essentially, we are using a specific prior over the factor loadings $w$ that allows us to prune away dimensions in the latent space. The prior is determined by a precision hyperparameter $\alpha$. Here, smaller values of $\alpha$ correspond to more important components.
+Essentially, we are using a specific prior over the factor loadings $\mathbf{W}$ that allows us to prune away dimensions in the latent space. The prior is determined by a precision hyperparameter $\alpha$. Here, smaller values of $\alpha$ correspond to more important components.
 You can find more details about this in e.g. [5].
 
 ```julia
 @model function pPCA_ARD(x, ::Type{TV}=Array{Float64}) where {TV}
-    # retrieve the dimension of input matrix x.
+    # Dimensionality of the problem.
     N, D = size(x)
 
     # latent variable z
-    z ~ filldist(Normal(), D, N)
+    Z ~ filldist(Normal(), D, N)
 
-    # Determine the number of loadings, i.e. number of columns of w, with Automatic Relevance Determination
-    alpha ~ filldist(Gamma(1.0, 1.0), D)
-    w ~ filldist(MvNormal(zeros(D), 1.0 ./ sqrt.(alpha)), D)
+    # weights/loadings w with Automatic Relevance Determination part
+    α ~ filldist(Gamma(1.0, 1.0), D)
+    W ~ filldist(MvNormal(zeros(D), 1.0 ./ sqrt.(α)), D)
 
-    mu = (w' * z)'
+    mu = (W' * Z)'
```

Comment:
This is just
```suggestion
    mu = Z' * W
```

Reply:
yes, this is from the original version - just changed it.